playwright-e2e/tests/dsm/participant/participant-phone-number-info-displayed.spec.ts

```diff
@@ -172,3 +160,32 @@ test.describe(`Confirm that participant phone number information is displayed @d
     })
   }
 })
+
+async function getDisplayedConsent(surveyDataTab: SurveyDataTab): Promise<Locator> {
+  const consentFormVersionOne = await surveyDataTab.getActivity({
+    activityName: SurveyName.RESEARCH_CONSENT_FORM,
```

Comment:
Is there a way to make the test pass regardless of version?  It doesn't look like we're doing different assertions based on version, and if we can make the test do what it needs to do without version, we won't have to change the test again for consent v4.

Reply:
good point - will make the change